api/plugins/notifications.py

```diff
@@ -36,21 +37,29 @@ def access_request_completed(
 
     @hookspec
     def access_expiring_user(
-        self, groups: list[OktaGroup], user: OktaUser, expiration_datetime: datetime.datetime
+        self, groups: list[OktaUserGroupMember], user: OktaUser, expiration_datetime: datetime.datetime
```

Comment:
Hmm, this and any other changes to existing hookspecs is going to be a breaking change to everyone's implementations of this notifications plugin. We should probably append new parameters/arguments instead of changing the type of existing ones.